content/module-reference/processing-modules/color-mapping.md

```diff
@@ -25,7 +25,7 @@ acquire as source/target
 : Press these buttons to generate color clusters for the source and target image, respectively. The processing takes a few seconds during which the GUI will be unresponsive.
 
 number of clusters
-: The number of color clusters to use. If you change this parameter all collected color clusters are reset and must be re-acquired.
+: The number of color clusters to use. It should correlate to the number of dominant colors in the image. The clusters are aquired through random sampling, so if you add more clusters in a complex image, there is bigger chance that the clusters picked out through chance are not appropriate. If you change this parameter all collected color clusters are reset and must be re-acquired.
```

Comment:
I would probably word it something like this:

The number of color clusters to use. It should roughly correlate to the main number of colours dominating the image. The clusters are required through a statistical sampling process to represent the colours occuring most commonly throughout the image. The random nature of the sampling process means that you might get varying results each time you aquire the clusters, especially if you have a larger number of clusters and/or a complex colour pallette in the image. If you change this parameter all the collected colour clusters are reset and must be re-acquired.

Reply:
@matt-maguire: What is the "**main** number of colours"? Presumably you mean "The clusters are **acquired** through..."?

Reply:
By "main number of colors" I mean "the number of dominant colors in the image"

Reply:
Ok, so "the main number of colors dominating the image" was basically saying "the number of dominant colors dominating the image" so my alternative wording is ok?